java/util/src/main/java/com/google/protobuf/util/JsonFormat.java

```diff
@@ -1126,145 +1158,159 @@ public int compare(final Object o1, final Object o2) {
       generator.print("}");
     }
 
-    private void printSingleFieldValue(FieldDescriptor field, Object value) throws IOException {
+    private void quoted(String value, boolean quote) throws IOException {
```

Comment:
This is only called once and the name doesn't really capture the purpose. Let's inline the logic for clarity.